examples/Makefile.include

```diff
@@ -14,8 +14,14 @@ else
   MSBUILD = $(shell which msbuild || (echo '\n\n\033[0;31mWarning:\033[0m could not find "msbuild", trying (deprecated) "xbuild"\n\n'>&2; which xbuild))
 endif
 
+# we ignore the return result in benchmark runs because we can have micro-benchmarks which
+# produce error asserts when executed with '--admit_smt_queries true'
```

Comment:
are there many of these? is it worth trying to fix at all? (probably not, I guess?)

Reply:
For some of them it could be reasonable behaviour, for others we might not be expecting an smt query. I'm not sure its the place of this patch to fix/investigate it. Let me know if filing an issue for tracking is useful or just noise?

Here is the full list of the micro-benchmarks that I see error with `--admit_smt_queries true`:
```
Unit2.fst(37,0-38,60): (Error 303) This top-level definition was expected to fail, but it succeeded
WPExtensionality.fst(117,0-118,34): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.BST.fst(37,0-37,42): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.BST.fst(40,0-40,65): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.BST.fst(43,0-43,87): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Heap.fst(26,0-26,54): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.ShortCircuiting.fst(21,0-21,33): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.ShortCircuiting.fst(25,0-25,36): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Bug260.fst(26,0-26,19): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(20,0-20,10): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(23,0-23,10): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(26,0-26,35): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(29,0-30,15): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(37,0-37,44): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(41,0-41,34): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(45,0-45,31): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(49,7-49,53): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Neg.fst(54,0-54,27): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(21,0-21,29): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(25,0-28,35): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(33,0-36,54): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(40,0-42,29): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(52,0-55,29): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(63,0-67,29): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(71,0-75,35): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(79,0-82,14): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(86,0-89,31): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Termination.fst(93,0-96,26): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.False.fst(23,0-23,33): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Set.fst(27,0-28,30): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Set.fst(32,0-33,67): (Error 303) This top-level definition was expected to fail, but it succeeded
NegativeTests.Set.fst(37,0-38,52): (Error 303) This top-level definition was expected to fail, but it succeeded
TestHasEq.fst(57,0-58,19): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(23,0-25,30): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(28,0-30,14): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(33,0-35,14): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(38,0-40,40): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(45,0-47,24): (Error 303) This top-level definition was expected to fail, but it succeeded
PatAnnot.fst(54,0-55,51): (Error 303) This top-level definition was expected to fail, but it succeeded
Test.FunctionalExtensionality.fst(36,0-36,50): (Error 303) This top-level definition was expected to fail, but it succeeded
Test.FunctionalExtensionality.fst(69,0-80,43): (Error 303) This top-level definition was expected to fail, but it succeeded
Test.FunctionalExtensionality.fst(83,0-92,52): (Error 303) This top-level definition was expected to fail, but it succeeded
Test.FunctionalExtensionality.fst(142,0-142,58): (Error 303) This top-level definition was expected to fail, but it succeeded
MustEraseForExtraction.fst(18,0-18,13): (Error 303) This top-level definition was expected to fail, but it succeeded
MustEraseForExtraction.fst(23,0-23,13): (Error 303) This top-level definition was expected to fail, but it succeeded
```